### PR TITLE
[Tokens refactoring #1] Remove tons of tokens lists recalculations by moving them to Updater

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

`useAllTokens()` used to calculate the tokens list on each subscription, now it does few times in `TokensListUpdater`

  # To Test

1. Test through https://github.com/cowprotocol/cowswap/pull/1822

### CODE REVIEW

File: src/cow-react/modules/tokensList/TokensListUpdater.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { useCombinedActiveList } from 'state/lists/hooks'
+import { useTokensFromMap } from '@src/hooks/Tokens'
```
Comment (alfetopito) [inline src/cow-react/modules/tokensList/TokensListUpdater.ts:2]: @ import?

```suggestion
import { useTokensFromMap } from 'hooks/Tokens'
```

**Review by alfetopito — APPROVED** [review]
Code looks great!

**Comment by elena-zh** [comment]
Hey @shoom3301 , could you please describe a bit more about the testing steps?
What options should I choose for the Lighthouse reports comparison?
Should I compare with Prod/Barn/Dev version?

**Comment by shoom3301** [comment]
Hi @elena-zh !
Please, test it through https://github.com/cowprotocol/cowswap/pull/1822. Thank you!

**Review by anxolin — COMMENTED** [review]
Awesome 🔥

**Pull request merged**